js/data/locale/my/dateformats.json

```diff
@@ -68,88 +68,88 @@
                 "h": "h",
                 "m": "mm",
                 "s": "ss",
-                "ah": "B h",
-                "hm": "B h:mm",
+                "ah": "a h",
+                "hm": "a h:mm",
                 "ms": "mm:ss",
-                "ahm": "hB h:mm",
-                "hms": "B h:mm:ss",
-                "hmz": "z B h:mm",
-                "ahmz": "z hB h:mm",
-                "ahms": "hB h:mm:ss",
-                "hmsz": "z B h:mm:ss",
-                "ahmsz": "z hB h:mm:ss"
+                "ahm": "a h:mm",
```

Comment:
The "B" should be there instead of "a" for any format that has "h" in it instead of "H". The conversion tool should handle it just like "a" format for other locales. We should not convert it to "a", which is only "AM" and "PM". Also, the conversion tool needs to extract the custom time periods from the CLDR data.

Reply:
My idea was  just remove "B" character in this PR at this point and just replace `a` as a temporarily fix (  because ilib is not ready to show that `dayperiod`  formatting style yet) I thought that's better than remaining `B` character here.
then when the  tool code extrac that data and ilib is ready to support that format style,  (will work different PR) that format will rollback.

Reply:
I see. Okay.